src/Components/Server/src/Circuits/CircuitSecretFactory.cs

```diff
@@ -5,20 +5,19 @@
 using System.Security.Cryptography;
 using Microsoft.AspNetCore.DataProtection;
 using Microsoft.AspNetCore.WebUtilities;
-using Microsoft.Extensions.Options;
 
 namespace Microsoft.AspNetCore.Components.Server.Circuits
 {
     // This is a singleton instance
     // Generates strong cryptographic ids for circuits that are protected with authenticated encryption.
-    internal class CircuitIdFactory
+    internal class CircuitSecretFactory
     {
         private const string CircuitIdProtectorPurpose = "Microsoft.AspNetCore.Components.Server";
```

Comment:
Can we change this to `Microsoft.AspNetCore.Components.Server.CircuitSecretFactory`? It's more scoped, and the constant name should also be updated.